public/views/goggles.html

```diff
@@ -0,0 +1,25 @@
+<div class="goggles-backgrounded goggles-xray-image">
+
+      <img src="/Shapes.jpg">
+
+      <div class="goggles-main-box goggles-box">
+        <h1>Remix this page</h1>
+        <ol class="goggles-listing">
+          <li><p><button class="goggles-activate-button">Activate X-ray Goggles</button></p></li>
+          <li><p>Tap the background image</p></li>
+          <li><p>Change the image's <i>source</i> from Shapes.jpg to Blur.jpg
```

Comment:
`'` should be `&rsquo;` (or `’`)
`<i></i>` should be `<em></em>` (`<i>`, when being used for “italics”, is deprecated in html5)